frontend/add-leads/add-edit-materialEntry.component.tsx

```diff
@@ -0,0 +1,70 @@
+import { useCss } from "kremling";
+import React from "react";
+import css from "./add-edit-materialEntry.css";
+
+export default function MaterialEntry({
+  materialProp,
+  handleDelete,
+  handleEdit,
+}) {
+  const [editing, setEditing] = React.useState(false);
+  const [material, setMaterial] = React.useState({ id: "", name: "" });
+
+  React.useEffect(() => {
+    setMaterial(materialProp);
+  }, []);
+
+  function OnValueChange(evt) {
```

Comment:
Component functions are uppercase, handler functions are lower case.

```suggestion
  function onValueChange(evt) {
```